index.html

```diff
@@ -0,0 +1,226 @@
+    main {
+      margin: auto;
+      width: 50%;
+      padding: 20px;
+      text-align: center;
+    }
+    h1 {
+      font-size: 3.5em;
+    }
+    </style>
+  </head>
+
+  <body>
+    <main>
+      <h1>TURN sample: Get a relay configuration and use it to stream</h1>
+      <p>Loading content from the API: <b id="message"></b></p>
+      <h1>Video Chat</h1>
+      <video id="local-video" height="150" autoplay></video>
+      <video id="remote-video" height="480" autoplay></video>
+    
+      <div>
+        <button id="get-video">Get Video</button>
+        <button id="call">Call</button>
+      </div>
+    </main>
+    <script src="https://cdn.socket.io/3.1.1/socket.io.min.js" integrity="sha384-gDaozqUvc4HTgo8iZjwth73C6dDDeOJsAgpxBcMpZYztUfjHXpzrpdrHRdVp8ySO" crossorigin="anonymous"></script>
+    <script type = "module">
+
+    const VideoChat = {
+        // Initialize our connection to the WebSocket.
+        socket: io(),
+      
+        // Call to getUserMedia (provided by adapter.js for cross browser compatibility)
+        // asking for access to both the video and audio streams. If the request is
+        // accepted callback to the onMediaStream function, otherwise callback to the
+        // noMediaStream function.
+        requestMediaStream: function(event){
+            navigator.mediaDevices.getUserMedia({video: true, audio: true})
+            .then(VideoChat.onMediaStream)
+            .catch(function(err) {
+                console.log(err.name + ": " + err.message);
+            });
+        },
+      
+        // The onMediaStream function receives the media stream as an argument.
+        onMediaStream: function(stream){
+          console.log('\nOnMedia\n');
+          
+          // Get the video element.
+          VideoChat.localVideo = document.getElementById('local-video');
+          
+          // Turn the volume down to 0 to avoid echoes.
+          VideoChat.localVideo.volume = 0;
+          VideoChat.localStream = stream;
+          console.log('\n Set attribute\n');
+          
+          // Turn the media stream into a URL that can be used by the video and add it as the video's `src`.
+          // As the video has the `autoplay` attribute it will start to stream immediately.
+          VideoChat.localVideo.srcObject = stream;
+          
+          // Now we're ready to join the chat room.
+          VideoChat.socket.emit('join', 'test');
+          console.log('\nready to call\n');
+          VideoChat.socket.on('ready', VideoChat.readyToCall);
+          console.log('\nOffer\n');
+          VideoChat.socket.on('offer', VideoChat.onOffer);
+          console.log('\nOnMedia end\n');
+        },
+      
+        // There's no media stream for this demo, so we will stop.
+        noMediaStream: function(){
+          console.log('No media stream for us.');
+          // Sad trombone.
+        },
+      
+        // When we are ready to call, enable the Call button.
+        readyToCall: function(event){
```

Comment:
prefer indicating unused args with a leading underscore.  ie,
``` function (_event)```

Reply:
same with a few of these: 
![image](https://user-images.githubusercontent.com/58831720/143143630-473ebe13-3f50-4bec-b0bd-19afa1a71cfd.png)